desktop/src/main/java/bisq/desktop/primary/main/content/wallet/config/WalletConfigPopup.java

```diff
@@ -85,7 +85,9 @@ public void onViewDetached() {
         }
 
         private void onConnectToWallet() {
-            String passphrase = model.walletPassphraseProperty.get();
+            String passphraseValue = model.walletPassphraseProperty.get();
+            Optional<String> passphrase = passphraseValue.isEmpty() ? Optional.empty()
+                    : Optional.of(passphraseValue);
             model.walletPassphraseProperty.setValue(""); // Wipe passphrase from memory
```

Comment:
Actually to avoid that the passphrase sticks in memory (until GC cleans up) we should use a char array instead of a String and clear the array afterwards. But no need to do it now, just for hardening once we get more production ready.